### ISSUE TITLE
sendGasless returns wrong transaction details

### ISSUE THREAD

**Issue by ArthurD1** [thirdweb-dev/js#1755]
Hello, 

while working on a gasless solution I had the following issue.
The transaction returned by this sendGasless function is errored.
Indeed most of the time the returned value is like.

```
`{
  chainId: "0xaa36a7",
  data: "0xdf905caf000000...",
  from: "0xaa185b756e2eecf44bc63217465865105d8f15d7",
  gas: "0x60822",
  gasLimit: "0x60822",
  gasPrice: "0x6fc23ad9",
  hash: "0xe7a460d1825658c57115cacc43f4af7fc0720cd24c80d0911fdf897cc5890add",
  input:
    "0xdf905caf0000000000000.......",
  maxFeePerGas: "0x6fc23ad9",
  maxPriorityFeePerGas: "0x6fc23ac0",
  nonce: "0xe",
  r: "0x7507203e467af0d1f65704d32ee251a6d8fcbbf9cd42...",
  s: "0x0",
  v: "0xecaaf84eca1c0d12933039118480913f8c2815df906e9ce3a544b7bb7ac9937b",
  to: "0xebe63b85b54bc280b3d4dab86064f3fb91aeb4c8",
  transactionIndex: null,
  type: "0x2",
  value: "0x0",
};`
```

Then the provider will try to format this value and return an error with code: `NUMERIC_FAULT`
This is because the provider is trying to format `v` as a BigNumber while the number is definitely out of range. (swapped with s?)

This is the code snippet that is at cause: 
https://github.com/thirdweb-dev/js/blob/dcdb0dcb838168a22c8335738852316dfb6e6fc9/packages/sdk/src/evm/core/classes/transactions.ts#L504-L538

While debugging this I noticed that after a a very short amount of time the returned transaction is okay!
My guess is that `this.provider.getTransaction(txHash)` returns some bad data in the first seconds (milliseconds maybe) and then start providing the right data. 

I am using a Defender Relayer with an autotask that calls a custom forwarder.
Basically the main contract is a custom signature mint contract. My thirdweb app will call the autotask thanks to a webhook and a transaction will be forwarded to the relayer.
I am testing this on `Sepolia`.
Contract aAddress : `0xF8145a33820288fC8efb93c565d8E3e79850a322`
Forwarder Address : `0xEBe63B85B54BC280B3D4dAb86064F3fB91AEb4C8`
Relayer Address: `0xaA185B756E2EECF44BC63217465865105d8F15D7`

Thank you for the help!

**Comment by joaquim-verges** [comment]
@ArthurD1 are you running the latest version of the SDK? 

asking because we added this try/catch:

```ts
try { 
       sentTx = await this.provider.getTransaction(txHash); 
     } catch (err) { 
       // some providers can throw an error if the tx is very recent 
     } 
```

for the exact reason you mention (relayed tx is not fully available yet, returning some partial response)

**Comment by ArthurD1** [comment]
Hi @joaquim-verges,

You are right:

```
        "@thirdweb-dev/react": "^3.16.5",
        "@thirdweb-dev/sdk": "^3.10.67",
```

Sorry for crying wolf and thank you for the prompt answer!
